library/pod-security-policy/apparmor/src.rego

```diff
@@ -0,0 +1,27 @@
+package k8spspapparmor
+
+violation[{"msg": msg, "details": {}}] {
+    metadata := input.review.object.metadata
+    not input_apparmor_allowed(metadata)
+    msg := sprintf("AppArmor profile is not allowed, pod: %v. Allowed profiles: %v", [input.review.object.metadata.name, input.parameters.allowedProfiles])
+}
+
+input_apparmor_allowed(metadata) {
+    metadata.annotations[key]
+    [prefix, name] = split_annotation(key)
+    prefix == "container.apparmor.security.beta.kubernetes.io"
+    annotation := sprintf("container.apparmor.security.beta.kubernetes.io/%v", [name])
```

Comment:
`startswith(annotion, "container.apparmor.security.beta.kubernetes.io/")` and avoids extra code

Reply:
thanks, updated. we still need to check for the container name, but this gets rid of some of the extra code